GameServerLib/Logic/Game.cs

```diff
@@ -129,42 +131,42 @@ public bool LoadScripts()
 
         public void NetLoop()
         {
-            _lastMapDurationWatch = new Stopwatch();
-            _lastMapDurationWatch.Start();
-
             while (!SetToExit)
             {
                 while (_server.Service(0, out var enetEvent) > 0)
                 {
                     switch (enetEvent.Type)
                     {
                         case EventType.Connect:
+                        {
                             // Set some defaults
                             enetEvent.Peer.Mtu = PEER_MTU;
                             enetEvent.Data = 0;
-                            break;
-
+                        }   
+                        break;
                         case EventType.Receive:
+                        {
                             var channel = (Channel)enetEvent.ChannelID;
                             PacketHandlerManager.HandlePacket(enetEvent.Peer, enetEvent.Packet, channel);
                             // Clean up the packet now that we're done using it.
                             enetEvent.Packet.Dispose();
-                            break;
-
+                        }
+                        break;
                         case EventType.Disconnect:
+                        {
                             HandleDisconnect(enetEvent.Peer);
-                            break;
+                        }
+                        break;
                     }
                 }
-
+                
                 if (IsPaused)
                 {
                     _lastMapDurationWatch.Stop();
                     _pauseTimer.Enabled = true;
                     if (PauseTimeLeft <= 0 && !_autoResumeCheck)
                     {
-                        PacketHandlerManager.GetHandler(PacketCmd.PKT_UNPAUSE_GAME, Channel.CHL_C2S)
-                            .HandlePacket(null, new byte[0]);
+                        PacketHandlerManager.GetHandler(PacketCmd.PKT_UNPAUSE_GAME, Channel.CHL_C2S).HandlePacket(null, new byte[0]);
```

Comment:
Not sure why this change is here since nothing seems to be changed, but it is not gonna work, use IPacketHandlerManager.UnpauseGame()
https://github.com/LeagueSandbox/GameServer/blob/828e0f7714a644ed2aff87a0364a57da5800efe0/GameServerCore/Packets/Handlers/IPacketHandlerManager.cs#